library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -179,7 +179,7 @@ input::placeholder {
 	color: var(--sakai-text-color-1);
 	font-family: $font-family;
 	font-size: #{$default-font-size - 1};
-	padding: 0.3em 2.2em 0.3em 0.5em;
+	padding: 0.3em 2.2em 0.3em 0.5em !important;
```

Comment:
`!important` is too strong on single-class selector that is used throughout Sakai. This could impact the spacing and alignment of other select elements, such as the multi-select panels and be hard to override later. 

I think a targeted approach to override the library's use of padding for the affect element would be better.